Tools/Scripts/webkitpy/layout_tests/models/test_configuration_unittest.py

```diff
@@ -244,6 +310,25 @@ def test_macro_expansion(self):
         ])
         self.assertEqual(converter.to_config_set(set(['win', 'mac', 'release'])), configs_to_match)
 
+        all_test_configurations = make_mock_all_test_configurations_set(('normal', 'asan', 'guard-malloc'))
+        converter = TestConfigurationConverter(all_test_configurations, MOCK_MACROS)
+        configs_to_match = set([
+            TestConfiguration('snowleopard', 'x86', 'release'),
+            TestConfiguration('snowleopard', 'x86', 'release', 'asan'),
+            TestConfiguration('snowleopard', 'x86', 'release', 'guard-malloc'),
+        ])
+        self.assertEqual(converter.to_config_set(set(['mac', 'release'])), configs_to_match)
+
+        configs_to_match = set([
+            TestConfiguration('lucid', 'x86_64', 'debug'),
+            TestConfiguration('lucid', 'x86_64', 'debug', 'asan'),
+            TestConfiguration('lucid', 'x86_64', 'debug', 'guard-malloc'),
+            TestConfiguration('lucid', 'x86_64', 'release'),
+            TestConfiguration('lucid', 'x86_64', 'release', 'asan'),
+            TestConfiguration('lucid', 'x86_64', 'release', 'guard-malloc'),
+        ])
+        self.assertEqual(converter.to_config_set(set(['x86_64'])), configs_to_match)
```

Comment:
Would it make sense to have a test where `'asan'` or `'guard-malloc'` art port of the config set?

Maybe like this?  (This is just a guess at whether this code is correct.)

```
        configs_to_match = set([
             TestConfiguration('lucid', 'x86_64', 'debug', 'asan'),
             TestConfiguration('lucid', 'x86_64', 'release', 'asan'),
         ])
         self.assertEqual(converter.to_config_set(set(['x86_64', 'asan'])), configs_to_match)
```

Reply:
These tests DO test that.